src/modules/app/StoreActionType.js

```diff
@@ -123,6 +123,10 @@ export type ClearCityContentActionType = {|
   type: 'CLEAR_CITY_CONTENT'
 |}
 
+export type ClearCityActionType = {|
```

Comment:
We need two actions because we need to call asynchronous methods (`clearCitySelection`) and modify the state. So one is a saga to call the async method and one a normal redux reducer.